ilastik/utility/exportFile.py

```diff
@@ -243,12 +245,34 @@ def _slice_range(low, high, max_):
     return slice(start, end, None)
 
 
-def create_slicing(axistags, dimensions, margin, feature_table):
+def create_slicing(
+    axistags: AxisTags, dimensions: Tuple[int, int, int, int, int], margin: int, feature_table: np.ndarray
+) -> Iterator[Tuple[List[slice], int]]:
     """
-    Returns an iterator on the slices for each object roi
+    Returns an iterator on the slices for each object (bounding box + margin)
```

Comment:
May as well use the `Returns:` section from Google Sphinx format.

Reply:
google styleguide even foresees a `Yields` keyword instead of `Returns` for generators